### DISCUSSION TITLE
[bug] "Got timeout reading communication packets." Error during install

### DISCUSSION THREAD

**Discussion by BasicComputers** [YetiForceCompany/YetiForceCRM#15121] [How to] [answered]
Attempting to install an instance of Yetiforce 4.4.0 on Windows Server 2019 using Xampp and connecting it to a Mysql Server 8.0 instance. When I go through the install process I am receiving an error that states "Got timeout reading communication packets."
Any idea what might be causing these issues with the setup, I have gone through the my.ini file that was included with the Yetiforce Xampp folder and copied those settings to the my.ini instance of Mysql Server 8.0 but I still receive the error message. 

![Yetiforce Error](https://user-images.githubusercontent.com/71788238/131405941-ce192f4d-5561-4b71-9030-d914a0b9f316.PNG)

**Comment by bpabiszczak** [comment]
YetiForce 4.4 :o
Windows Server :o

Check this: https://youtu.be/IUB-wjXUREE

**Reply by BasicComputers** [reply]
Yetiforce 4.4 is the last version offered with Xampp for Windows and I planned on upgrading once the initial install was done. I had gone and replaced the htdocs with Yetiforce 5.2.0 as well and received the same error so a different version does not make a difference. I had this same configuration working on a Windows Server 2012 box much earlier but am encountering an error when setting up on the Windows Server 2019 OS, Just looking to see if there is an answer that is known for this error, no need to ridicule the fact that it is on an older version and using Xampp on Windows which was previously an option with the yetiforce packages.

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
xampp_YetiForceCRM-4.4.0-complete.zip the package was made at the request of the client (paid for it).
version 4.4 is not compatible with MySql 8.0
Use the latest version of CRM and run Xampp according to our requirements and it will work

**Reply by BasicComputers** [reply]
Thank you for your help, I downloaded the latest version of Xampp with PHP 7.4 bundled and was able to get the Mysql database to connect after updating the php configuration as required.
